lib/companies/pending_changes.ex

```diff
@@ -47,62 +30,66 @@ defmodule Companies.PendingChanges do
     end
   end
 
-  defp apply_changes("create", changeset), do: Repo.insert(changeset)
-  defp apply_changes("update", changeset), do: Repo.update(changeset)
+  def create(changeset, action, user, note \\ "")
```

Comment:
Why did you move these? They were in alphabetical order before.

Reply:
Sorry, I didn't notice that you had them in alphabetical order and I moved them around in a way that made sense to me (and, I thought it might be in the longs run). I moved the api methods on top and the private below, moving the easy to deal with methods on the far bottom.